plugins/explore/src/extensions.tsx

```diff
@@ -25,3 +28,36 @@ export const ExplorePage = explorePlugin.provide(
     mountPoint: exploreRouteRef,
```

Comment:
This ends up being a bit lost when the page is completely defined in the app. Are you in the mood for hooking this up with `useOutlet` so that the code ends up something like this in the app instead?

```tsx
<FlatRoutes>
  <Route path="/explore" element={<ExplorePage/>}>
    {explorePage}
  </Route>
</FlatRoutes>
```

Best reference for this is how the [`SearchPage`](https://github.com/backstage/backstage/blob/f07426d3ab332f42c4686eef6e99028b03965a6e/plugins/search/src/components/SearchPage/SearchPage.tsx#L47) works

Reply:
Yep, I can also do that. I actually experimented with that pattern but it ended up just being a simple pass through so it felt a bit unnecessary. However the current approach requires the component instance to be passed in or the explore plugin needs to be configured in `packages/app/src/plugins.ts` to avoid an API factory error. I supposed following the outlet pattern avoids this as well.

Reply:
@Rugvip - I've refactored the ExplorePage to use a route `outlet` and moved the existing default layout to a new `DefaultExplorePage` which is not exported outside the page. The `ExplorePage` is now super basic, but it does create more consistency with how SearchPage works.